VoxelSniperCore/src/test/java/com/github/kevindagame/voxelsniper/BrushesTest.java

```diff
@@ -149,4 +152,20 @@ public void testPerformerBrushesArgumentValuesOverloading() {
         }
         System.out.println("Performer Arguments VALUES Test OK!");
     }
+
+    @Test
+    public void testBrushPermissions() {
+        // Load all brushes and permissions
+        VoxelBrushManager brushes = VoxelBrushManager.initialize();
+        PermissionLoader permissions = PermissionLoader.getInstance();
+        permissions.load();
+
+        // check that every brush has a permission and that permission is registered in the permissions.yml file
+        for (Map.Entry<String, BrushData> e : brushes.getRegisteredBrushesMap().entrySet()) {
+            BrushData brushData = e.getValue();
+            Assert.assertNotNull("Brush permission must not be null (brush: " + brushData.getName() + ")", brushData.getPermission());
+            Permission permissionNode = permissions.getPermission(brushData.getPermission());
+            Assert.assertNotNull("Brush permission must be in permissions.yml (brush: " + brushData.getName() + ")", permissionNode);
+        }
+    }
```

Comment:
Could we also add a test to see if every permission in the yml is used?

Reply:
Done, also removed some old permissions and found some other small bugs